discovery-infra/test_infra/controllers/node_controllers/libvirt_controller.py

```diff
@@ -96,8 +109,124 @@ def format_disk(disk_path):
         # Fix for libvirt 6.0.0
         if image_size.isdigit():
             image_size += "G"
-        command = f'qemu-img create -f qcow2 {disk_path} {image_size}'
-        utils.run_command(command, shell=True)
+
+        cls.create_disk(disk_path, image_size)
+
+    @staticmethod
```

Comment:
What's the benefit of having all these functions as `staticmethod`s and also at the same time private functions? Isn't this contradictional?

Reply:
`@staticmethod` technically only makes the calling requirements less strict, so if a method *can* be static, it *should* be static. PyCharm even detects methods that don't use self and suggests you convert them to static, rightfully so in my opinion.

One argument for why you would want to make a private method static - Say you want to add a future, public method to the class, that actually *needs* to be static (for whatever reason). That method might want to make use of one of my private methods. If my methods are not static, that public static method cannot make use of them.